.github/workflows/licensed_cache_update.yml

```diff
@@ -0,0 +1,27 @@
+name: Licensed Cache Update
+
+on:
+  push:
+    branches:
+      - main
+      - licensed_cache_update
+    paths: package-lock.json
+  pull_request:
+    branches: main
+    paths: package-lock.json
+  workflow_dispatch:
+
+jobs:
+  licensed_cache_update:
+    runs-on: ubuntu-latest
+    name: Check licenses
+    steps:
+      - uses: actions/checkout@v1
+      - uses: jonabc/setup-licensed@v1.0.2
+        with:
+          version: '2.x'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+      - run: npm install --production
+      - uses: jonabc/licensed-ci@v1.5.3
```

Comment:
Consider pinning to v1 unless you have a specific reason to do this

Reply:
Agree!